packages/x-components/src/x-modules/facets-next/service/facets.service.ts

```diff
@@ -35,7 +40,7 @@ export class BaseFacetsService implements FacetsService {
     this.createEntity(filter).deselect(filter);
   }
 
-  saveFacets(facetGroup: FacetGroup): void {
+  updateFacets(facetGroup: FacetsGroup): void {
     const previousFilters = this.removeGroupFilters(facetGroup.id);
     facetGroup.facets.forEach(facet => {
```

Comment:
`facetsGroup.facets.forEach(facet => {`

Reply:
I got it, I forget to rename this variable everywhere, have mercy with this poor Sotrondian.